src/requests.js

```diff
@@ -1,11 +1,22 @@
 import axios from "axios";
 
-export const getReviews = () => {
-  return axios
-    .get("https://wandering-pink-gloves.cyclic.app/api/reviews")
-    .then((reviews) => {
-      return reviews.data;
-    });
+export const getReviews = (order, property, category) => {
```

Comment:
This could make use of axios params, think it would make your code more readable.